<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="DocumentView" name="document_widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_saveas"/>
    <addaction name="separator"/>
    <addaction name="action_properties"/>
    <addaction name="separator"/>
    <addaction name="action_print"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_users">
    <property name="title">
     <string>&amp;Users</string>
    </property>
    <addaction name="action_newuser"/>
    <addaction name="action_removeuser"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_users"/>
   <addaction name="menu_view"/>
   <addaction name="menu_window"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="file_toolbar">
   <property name="windowTitle">
    <string>File</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="separator"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="document_toolbar">
   <property name="windowTitle">
    <string>Document</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_properties"/>
  </widget>
  <widget class="QDockWidget" name="games_tree_dock_widget">
   <property name="windowTitle">
    <string>Games</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="games_tree_dock_widget_contents">
    <layout class="QVBoxLayout" name="games_tree_dock_widget_layout">
     <item>
      <widget class="GamesTreeWidget" name="games_tree_widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>&amp;New…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>&amp;Open…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>&amp;Save…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_saveas">
   <property name="text">
    <string>Save As…</string>
   </property>
  </action>
  <action name="action_newuser">
   <property name="text">
    <string>Add a new user…</string>
   </property>
  </action>
  <action name="action_removeuser">
   <property name="text">
    <string>Remove a user…</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_properties">
   <property name="text">
    <string>Properties…</string>
   </property>
  </action>
  <action name="action_print">
   <property name="text">
    <string>&amp;Print…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DocumentView</class>
   <extends>QWidget</extends>
   <header>genial.views.documentview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GamesTreeWidget</class>
   <extends>QWidget</extends>
   <header>genial.gamestreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/genial.qrc"/>
 </resources>
 <connections/>
</ui>
